addons/event/static/src/client_action/event_barcode.js

```diff
@@ -19,6 +19,7 @@ export class EventScanView extends Component {
     setup() {
         this.actionService = useService("action");
         this.dialog = useService("dialog");
+        this.home = useService("home_menu");
```

Comment:
This service is only supported in Enterprise if I get it right. Some changes to do here (probably reintroducing onClickBackToEvents)